react/ContactsListModal/ContactsListContent.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { Contact } from 'cozy-doctypes'
+
+import ContactsList from '../ContactsList'
+import Spinner from '../Spinner'
+import EmptyMessage from './EmptyMessage'
+
+const mkFilter = filterStr => contacts => {
```

Comment:
could we rename that method please? i have no idea what mk could be 🙏

Reply:
c'est du code existant déplace en l'état, j'ai déjà passé du temps non estimé à faire de la refacto, je pense qu'on respecte déjà bien la boyscout rule ici :)